backend/services/battle.py

```diff
@@ -0,0 +1,85 @@
+from users.models import User
```

Comment:
As I mentioned before all helpers that start with `verify` should start with `has` or `is`

Reply:
thank you :)